service/src/main/java/org/broadinstitute/listener/relay/http/ListenerConnectionHandler.java

```diff
@@ -90,7 +90,7 @@ public Flux<RelayedHttpListenerContext> receiveRelayedHttpRequests() {
   public Mono<String> openConnection() {
     return Mono.create(
         sink -> {
-          logger.debug("Opening connection to Azure Relay.");
+          logger.info("Opening connection to Azure Relay.");
```

Comment:
Ah, that is not my intent. I thought this was only called at startup, when the listener attaches to the Relay, and not for each individual request from end users. Since this PR is focused on that startup/attachment, I intended to increase logging at that point only.

Reply:
@cpate4 I `kubectl`-ed into an existing listener deployment (i.e. prior to this PR) and manually set `LOGGING_LEVEL_ORG_BROADINSTITUTE_LISTENER` to `DEBUG`. Upon restart of that listener, I see these lines logged at startup:
```
2024-01-26 18:17:36.745   INFO 1 --- [  restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8080 (http) with context path ''
2024-01-26 18:17:36.941   INFO 1 --- [  restartedMain] o.b.listener.ListenerApplication         : Started ListenerApplication in 38.997 seconds (JVM running for 45.53)
2024-01-26 18:17:37.437   INFO 1 --- [  restartedMain] o.b.l.relay.http.AvailabilityListener    : LivenessState: CORRECT
2024-01-26 18:17:37.443  DEBUG 1 --- [  restartedMain] o.b.l.r.t.RelayedRequestPipeline         : Registering HTTP pipeline
2024-01-26 18:17:37.838  DEBUG 1 --- [  restartedMain] o.b.l.r.t.RelayedRequestPipeline         : Registering WebSocket upgrades pipeline
2024-01-26 18:17:37.840  DEBUG 1 --- [  restartedMain] o.b.l.r.http.ListenerConnectionHandler   : Opening connection to Azure Relay.
```
but I do _not_ see them repeated for each request. Does this change your stance?

Reply:
I think I said it, "will run for every connection being opened", not for every request.

That being said, I am slow (it's Friday) - I believe this listener is specific to a single user (i.e. meaning we are deploying one of these for each data plane set of apps, not sharing an AzureRelayListener, and not having to pull apart logs to separate out my requests from your requests) - so, yeah, should be good on the DEBUG/INFO level.

That being said, happy to leave them, and if we get to a point where they are noisy, we can handle it then. Thank you for entertaining me on this! - 🤗

Reply:
Sorry for the misunderstanding about connections vs requests! Cool, I will forge ahead with this and we can always pull it back if it's noisy, as you said.